collated/test/A0139772U.md

```diff
@@ -0,0 +1,72 @@
+# A0139772U
+###### \java\seedu\whatnow\logic\LogicManagerTest.java
+``` java
+    @Test
+    public void execute_exit() throws Exception {
+        assertCommandBehavior("exit", ExitCommand.MESSAGE_EXIT_ACKNOWLEDGEMENT);
+    }
+
+    @Test
+    public void execute_clear() throws Exception {
+        TestDataHelper helper = new TestDataHelper();
+        model.addTask(helper.generateTask(1));
+        model.addTask(helper.generateTask(2));
+        model.addTask(helper.generateTask(3));
+
+        assertCommandBehavior("clear", ClearCommand.MESSAGE_SUCCESS, new WhatNow(), Collections.emptyList());
+    }
+
+
+    @Test
+    public void execute_add_invalidArgsFormat() throws Exception {
+        String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
+        assertCommandBehavior(
+                "add wrong/args wrong/args", expectedMessage);
+    }
+
+    @Test
+    public void execute_add_invalidTaskData() throws Exception {
```

Comment:
test name should also contain the expected outcome. recollect that the test name has 3 parts.. blah... blah..